components/molecules/FeedbackWidget.js

```diff
@@ -195,40 +193,43 @@ export const FeedbackWidget = ({
                 ""
               )}
               <div className="layout-container text-gray-dark-100 pb-4">
-                <button
+                <ActionButton
                   id="feedbackClose"
+                  ariaLabel="Close the expanded feedback section"
+                  dataCy="closeButton"
+                  dataTestId="closeButton"
+                  custom="flex float-right pt-4 font-body text-gray-dark-100 flex mt-2.5 lg:mt-0 outline-none focus:outline-white-solid items-center"
```

Comment:
The focus outline on the close button doesn't look that great. A work around here would be to get rid of the `pt-4` from the custom attribute and instead put this ActionButton inside a div with top padding. 

ie.
```JSX
              <div className="layout-container text-gray-dark-100 pb-4">
                <div className="pt-2">
                <ActionButton
                  id="feedbackClose"
                  ariaLabel="Close the expanded feedback section"
                  dataCy="closeButton"
                  dataTestId="closeButton"
                  custom="flex float-right font-body text-gray-dark-100 flex mt-2.5 lg:mt-0 outline-none focus:outline-white-solid items-center"
                  imageSource="/close-x.svg"
                  imageAlt="Close button"
                  imageSpanClass="text-xs leading-4 lg:text-sm underline ml-2 lg:leading-10"
                  imageSpanText={t("close")}
                  onClick={() => {
                    toggleForm();
                    setCount(2000);
                  }}
                />
                </div>
```


<img width="555" alt="Screenshot 2022-02-02 151703" src="https://user-images.githubusercontent.com/71025360/152230483-348842e6-d4bc-4ec7-98cb-4089cfe449f6.png">